src/sim-host/ui/custom-elements.js

```diff
@@ -262,6 +266,10 @@ function initialize(changePanelVisibilityCallback) {
                 // Reverse the order of the checkbox and caption
                 this.shadowRoot.appendChild(this.shadowRoot.querySelector('label'));
             }
+            var shouldReadOut = this.getAttribute('read-out');
+            if (shouldReadOut && shouldReadOut == 'true') {
```

Comment:
You should implement this as a standard boolean attribute. That means you specify it without a value:

    <cordova-label read-out label="No values saved">

And you should be able to check for it simply using `this.hasAttribute('read-out')`